app/api-v1/routes/recipe/{id}.js

```diff
@@ -1,8 +1,15 @@
+const { ItemNotFoundError } = require('../../../utils/errors')
 // eslint-disable-next-line no-unused-vars
 module.exports = function (recipeService) {
   const doc = {
     GET: async function (req, res) {
-      res.status(500).json({ message: 'Not Implemented' })
+      const { id } = req.params
+      const { statusCode, result } = await recipeService.getRecipeByID(id)
+      if (result == {}) {
+        throw new ItemNotFoundError({ message: 'Item not found', service: 'order' })
```

Comment:
As in my other comment the `ItemNotFoundError` should probably be thrown in `getRecipeByID`. Also, and I get it was probably a typo, but watch out for double equals `==` checks (they trigger me horribly 😖)